### PR TITLE
Fixing various documentation typos 

### PR DESCRIPTION
**Pull request by lreis**
Closes [#4826 - @internationalized documentation typos](https://github.com/adobe/react-spectrum/issues/4826)

## ✅ Pull Request Checklist:

- [x] Included link to corresponding [React Spectrum GitHub Issue](https://github.com/adobe/react-spectrum/issues).
- [ ] Added/updated unit tests and storybook for this change (for new code or code which already has tests).
- [ ] Filled out test instructions.
- [x] Updated documentation (if it already exists for this component).
- [ ] Looked at the Accessibility Practices for this feature - [Aria Practices](https://www.w3.org/TR/wai-aria-practices-1.1/)

## 🧢 Your Project:

Adobe Globalization team - internal I18n audit

### CODE REVIEW

**Review by reidbarber — APPROVED** [review]
Thanks!

**Comment by yihuiliao** [comment]
GET_BUILD

**Comment by ktabors** [comment]
GET_BUILD

File: packages/@internationalized/date/docs/ZonedDateTime.mdx [hunk 1]
```diff
@@ -121,7 +121,7 @@ See the [Calendar](Calendar.html#implementations) docs for details about the sup
 
 Many calendar systems have only one era, or a modern era and a pre-modern era (e.g. AD and BC in the Gregorian calendar). However, other calendar systems may have many eras. For example, the Japanese calendar has eras for the reign of each Emperor. `ZonedDateTime` represents eras using string identifiers, which can be passed as an additional parameter to the constructor before the year. When eras are present, years are numbered starting from 1 within the era.
 
-This example creates a date in the Japanese calendar system, which is equivalent to April 4th, 2020 at 9:15 AM in the Gregorian calendar.
+This example creates a date in the Japanese calendar system, which is equivalent to April 30th, 2019 at 9:15 AM in the Gregorian calendar.
```
Comment (ktabors) [inline packages/@internationalized/date/docs/ZonedDateTime.mdx:124]: Should this have the year?

**Review by devongovett — APPROVED** [review]

File: packages/@internationalized/date/docs/ZonedDateTime.mdx (hunk 1 above)
Comment (lreis) [reply packages/@internationalized/date/docs/ZonedDateTime.mdx:124]: > Should this have the year?

This is the value returned by the sample code that follows it.

**Pull request merged**